charts/agent/scripts/appchecks2helm

```diff
@@ -1,11 +0,0 @@
-#!/bin/bash
```

Comment:
there is nothing else in that /scripts directory other than the removed appchecks2helm file. shall I remove the directory altogether? I assume there is no code scanning through the subdirectory structures and so it is safe to do.

Reply:
git doesn't keep empty directories, so since this was the only file under this folder it counts as deleting the folder too

Reply:
thanks! this is great to know.